clickstream-event-visualiser-ui/src/main/java/clickstream/event_visualiser_ui/ui/fragment/CSEvHomeListFragment.kt

```diff
@@ -0,0 +1,77 @@
+package clickstream.event_visualiser_ui.ui.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.TextView
+import androidx.core.widget.doOnTextChanged
+import androidx.lifecycle.lifecycleScope
+import androidx.recyclerview.widget.RecyclerView
+import clickstream.event_visualiser_ui.databinding.FragmentCsEvHomeBinding
+import clickstream.event_visualiser_ui.ui.CSEvListItem
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.launch
+
+internal class CSEvHomeListFragment : CSEvBaseListFragment<FragmentCsEvHomeBinding>() {
+
+    override val bindingProvider: (LayoutInflater) -> FragmentCsEvHomeBinding
+        get() = { li ->
+            FragmentCsEvHomeBinding.inflate(li).also {
```

Comment:
similar to https://github.com/gojek/clickstream-android/pull/17/files#r903366590

Reply:
Just once in `onCreateView` of `CSEvBaseListFragment`